src/import/pull-common.c

```diff
@@ -513,17 +516,11 @@ int pull_verify(ImportVerify verify,
                 return log_error_errno(SYNTHETIC_ERRNO(EBADMSG),
                                        "Checksum is empty, cannot verify.");
 
-        r = verify_one(checksum_job, main_job);
-        if (r < 0)
-                return r;
-
-        r = verify_one(checksum_job, roothash_job);
-        if (r < 0)
-                return r;
-
-        r = verify_one(checksum_job, settings_job);
-        if (r < 0)
-                return r;
+        FOREACH_POINTER(j, main_job, settings_job, roothash_job, roothash_signature_job, verity_job) {
```

Comment:
I doesn't matter too much, but it'd be nicer to list the variables here in the same order as in the function signature...

Reply:
it's the same order, just the checksum and the signature job are not listed, since they are what we verify things with, hence nothing have to verify...

Reply:
Indeed. I got confused by the diff.